### ISSUE TITLE
Latest version (3.2.4) is brokin on Node v6

### ISSUE THREAD

**Issue by bvanderlaan** [Surnet/swagger-jsdoc#144]
In the latest version this line (https://github.com/Surnet/swagger-jsdoc/blob/master/lib/helpers/hasEmptyProperty.js#L7) has resulted in the package no longer working in Node v6

This is because the API `Object.values` is not a valid function under Node v6 thus we get an Error when trying to use the package.

I'm thinking we can either change it to 
```
return Object.keys(obj).every(key => {
   const keyObject = obj[key];
   return typeof keyObject === 'object' &&
      Object.keys(keyObject).every(key => !(key in keyObject))
});
```
Or revert the last update and re-release it as a major change.

**Comment by kalinchernev** [comment]
Hi @bvanderlaan and thanks for submitting this issue.

Feel free to submit a pull request with your suggestion. There are tests to give fast feedback whether the change would achieve same result.

It will be a bugfix release.

**Comment by a-morn** [comment]
@bvanderlaan https://github.com/Surnet/swagger-jsdoc/pull/145

**Comment by kalinchernev** [comment]
@bvanderlaan @a-morn was very kind to submit a correction to the issue you reported. It's been merged and published under 3.2.5, please update.

**Comment by bvanderlaan** [comment]
👍 That did the trick. Thanks
